Installation/changes.html

```diff
@@ -1,4 +1,4 @@
-﻿<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
On my machine, this commit starts with:
```patch
diff --git a/Installation/changes.html b/Installation/changes.html
index 6fa29bf..a6cd54a 100644
--- a/Installation/changes.html
+++ b/Installation/changes.html
@@ -1,4 +1,4 @@
-<U+FEFF><!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <!-- $Id$
      $URL$
 -->
```

Your editor has removed the [BOM]. That is fine. It is optional anyway. However, your editor is not so polite, because it should not remove it when it is in the file.

  [BOM]: https://en.wikipedia.org/wiki/Byte_order_mark "Byte order mark"